web_environment_ribbon/static/src/js/ribbon.js

```diff
@@ -1,35 +1,57 @@
-/******************************************************************************
-    Copyright (C) 2015 Akretion (http://www.akretion.com)
-    @author Sylvain Calador <sylvain.calador@akretion.com>
+/* Copyright 2015 Sylvain Calador <sylvain.calador@akretion.com>
+   Copyright 2015 Javi Melendez <javi.melendez@algios.com>
+   Copyright 2016 Antonio Espinosa <antonio.espinosa@tecnativa.com>
+ * License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). */
 
-    This program is free software: you can redistribute it and/or modify
-    it under the terms of the GNU Affero General Public License as
-    published by the Free Software Foundation, either version 3 of the
-    License, or (at your option) any later version.
+(function ($) {
```

Comment:
Hmm yeah seems that just hasn't been updated; the v9 API uses a new requires system, which makes essentially all v8 JS incompatible.

The v9 API is described in the Odoo docs, the important portion starting at the [Module Overview](https://www.odoo.com/documentation/9.0/reference/javascript.html#javascript-module-system-overview) & an example can be seen [in website_field_autocomplete](https://github.com/OCA/website/blob/9.0/website_field_autocomplete/static/src/js/field_autocomplete.js#L5)